app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,17 @@
   </head>
 
   <body>
+
+    <% if notice %>
```

Comment:
This is devise generated, and there are 4 possible outcomes, not two. Don't touch this.